src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -620,10 +796,14 @@ end subroutine get_box
    !> \details 
    !
    !-----------------------------------------------------------------------
-   real (kind=RKIND) function get_var()
+   real (kind=RKIND) function get_var(box, box_mean, nx, ny)
 
       implicit none
 
+      real (kind=RKIND), dimension(:,:), pointer, intent(in) :: box   ! Subset of topography covering a grid cell
```

Comment:
It might be nice to be consistent with the use of `intent` for pointer arguments. In some new code in this PR, `intent` is omitted (defaulting to `inout`), and in some place an `intent` is declared.

Reply:
Thanks. I've added explicit intents for the other pointers that occur as dummy vars/arguments.